benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1150,6 +1150,8 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "requestedHearingsTextList", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "releasedHearingsTextList", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "requestingParty", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
-  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "showRequestingPartyPage", "UserRole": "caseworker-sscs", "CRUD": "CRUD"}
-
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "showRequestingPartyPage", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "showPostponementDetailsPage", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "showPostponementDetailsPage", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "postponementRequestDetails", "UserRole": "caseworker-sscs", "CRUD": "CRUD"}
```

Comment:
Everything inherits from `caseworker-sscs` so you could remove the clerk and superuser change if you want

Reply:
The idea was to hide out the components from DWP.

Reply:
but the DWP inherit from caseworker-sscs so they'd get this permission anyway. It's fine they have permission to this, just as long as it's hidden from the tabs.